tests/devhub/test_addon_submissions.py

```diff
@@ -35,5 +55,12 @@ def test_delete_all_extensions(selenium, base_url):
     we can start over with this user in the following runs and also for
     verifying that the addon deletion process functions correctly"""
     page = DevHubHome(selenium, base_url).open().wait_for_page_to_load()
-    # TODO: add steps for deleting the user addons
-    # once the test steps are complete, the 'clear_session' fixture will handle the clean-up part
+    manage_addons = page.click_my_addons_header_link()
+    # run the delete steps until all the addons are cleared from the list
+    while len(manage_addons.addon_list) > 0:
+        addon = manage_addons.addon_list[0]
+        edit = addon.click_addon_name()
+        manage = edit.click_manage_versions_link()
+        delete = manage.delete_addon()
+        delete.input_delete_confirmation_string()
+        delete.confirm_delete_addon()
```

Comment:
Can you put this in a callable fixture and not as a "test"?

Reply:
Well, even if this looks like a clean-up routine, it is also a proper test. We test that extensions can be deleted, so I'd rather have it in the test suite, to make it clear that we are testing add-on deletions.